WIP: Added blogpost about how to setup mailcow + mailadm (#604)
deltachat/deltachat-pages: _posts/2023-01-26-upcoming-mail-server-workshops.md

---

> title: New Guide for Setting up Mail Servers for Delta Chat - and Upcoming Workshops!

Reviewer: ```suggestion
title: New Mail Server Setup Guide for Delta Chat - and Upcoming Workshops!
```

Author: hm, not a fan... verbs > substantives. "mail server setup guide" is technically more correct, but as a reader it's harder to wrap your mind around it. I'm a bit nitpicky about this, but headlines are important^^

author: missytake, brabo
image: ../assets/blog/mailcow-UI-login.png
---

Every once in a while we get asked what mail server is best suited for Delta

> Chat. In short - most mail servers will work well, and it depends more on your
> IP reputation and configuration parameters like spam checking, rate limits, and

Reviewer: ```suggestion
Chat. In short - most mail servers will work well, and reliability depends more on their
IP reputation and configuration parameters like spam checking, rate limits, and
```

> attachment size how well it works with Delta Chat. But the mail server we found
> the easiest to manage is [Mailcow](https://mailcow.email).

Reviewer: ```suggestion
attachment size how well it works with Delta Chat. 
One feature-complete mail-server we have successfully worked with successfully is  [Mailcow](https://mailcow.email).
```

Mailcow takes some long-standing Free Software tools like postfix, dovecot, and
rspamd, wraps them into a present through docker, and ties a ribbon around it
with a neat web interface which abstracts many complex config parameters away

> from you. This way, you can't do much damage with a misconfigured postfix; but

Reviewer: ```suggestion
from you. This way, you can't easily end up with a misconfigured postfix; but
```

if you still need to tweak postfix for a specfic use case, you can manually
edit the postfix config files in the docker container.

> We are quite amazed by Mailcow and even changed our
> [mailadm](https://mailadm.readthedocs.io/) account creation tool to work with
> mailcow instead of postfix/dovecot. Mailadm makes it easy to create accounts
> for Delta Chat, the users just need to scan one QR code and are logged in.

Reviewer: ```suggestion

We implemented an automated e-mail account management tool
to help admins give out accounts to users, 
both random short-lived ("burner") or longer-living accounts. 
[Mailadm](https://mailadm.readthedocs.io/) is implemented in Python 
and uses a minimal part of the [ReST HTTP API of mailcow](https://mailcow.docs.apiary.io/#).
Mailadm allows an admin group to create and hand out "tokens" (URLs with a secret number) 
to users who just need to scan one QR code and are logged in automatically.
```
let's not go overboard with praise here and not single out mailcow too much. Other solutions could also work nice, so let's just be descriptive here.

> And now we want to help others with hosting this combination of tools. Today,
> we published [a guide on this website](serverguide), which you can use to setup
> a mail server of your own, together with mailadm as easy account creation tool
> to bring your friends and family to Delta Chat. [Check it out](serverguide),
> and [send us your feedback](mailto:mailadm@testrun.org) about it!

Reviewer: ```suggestion
With this blog post, we also publish  [a new guide](serverguide), which you can use to setup
a mailcow  server of your own, together with mailadm as easy account creation tool
to bring your friends and family to Delta Chat. 
We are happy about  [any feedback](mailto:mailadm@testrun.org) 
about our setup guidance and mailadm installation instructions!
```

Author: I wrote something similar?

> ## Upcoming Workshops

Reviewer: ```suggestion
## Upcoming mailcow/mailadm setup sessions in Amsterdam, Bruxelles, Montpellier and Gent
```

Author: I wrote something similar :)
